### DISCUSSION TITLE
ESP NOW Broadcast Mode

### DISCUSSION THREAD

**Discussion by gary7530** [pschatzmann/arduino-audio-tools#606] [Q&A] [answered]
Is there such a function?

```
const char *peers[] = { "FF:FF:FF:FF:FF:FF" };
auto cfg = now.defaultConfig();
now.begin(cfg);
now.addPeers(peers);
```

**Comment by pschatzmann** [comment]
I don't understand your question! Why don't you just look at the documentation ?

**Reply by gary7530** [reply]
Use ESP NOW without knowing the MAC address of other boards
I am on the send ` now.addPeer("FF:FF:FF:FF:FF:FF");`
But the receiver does not respond

**Reply by pschatzmann** [reply]
To my knowledge you need to provide the correct peer address!
Why do you think this should work ? Can you point to some official documentation form Espressif ?

**Reply by gary7530** [reply]
https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-reference/network/esp_now.html

**Reply by pschatzmann** [reply]
I don't see anywere that you can use this as valid peer address. 
Maybe you want to use UDP instead ?

**Reply by gary7530** [reply]
https://randomnerdtutorials.com/esp-now-auto-pairing-esp32-esp8266/

**Comment by pschatzmann** [comment, marked as answer]
That's a custom application on top of the esp-now protocol. 
Feel free to extend the existing functionality!
